### DISCUSSION TITLE
Quality Crops Bundle

### DISCUSSION THREAD

**Discussion by waynn** [nathan-alden-sr/stardew-valley#43] [Ideas]
It feels like there should be a mention of that in the summer 1 guide since you have to get 5 gold melons. 

@nathan-alden-sr

**Comment by nathan-alden-sr** [comment]
In general I avoided adding guidance for things outside the scope of just earning as much G as possible. You'll notice that Zamiell's guide mentions that a particular day in Spring is the last day you can plant certain Flowers, while my guide doesn't mention that. That's intentional; I wanted to leave non-G challenges up to the player as I found enjoyment myself having to track my own challenges.

That said, I have considered making a sort of "sister" guide that specifically tackles completing the Community Center and also Collections. The problem is those stupid Red Cabbage Seeds. I have no idea why ConcernedApe decided to make _one single crop_ not appear until Year 2--and that crop is needed in a Bundle! I know about the "guarantee you have a chance of acquiring a Red Cabbage Seed in Year 1" option at game creation but IMO this is a lame change that never should've happened. He should've simply made the crop available in Year 1, even if Sandy sells it. Even with the option selected, you have to make lengthy trips every Friday and Sunday to the Traveling Cart, and you may not even see the Red Cabbage Seeds until the very end of the year.

In my previous playthrough, I got lucky and acquired several Red Cabbage Seeds from various Monsters in Skull Cavern, but on my current playthrough, despite having every Monster-hunting goal completed by Fall, I didn't get a single drop. RNG sucks.

**Reply by waynn** [reply]
Totally get it, but I think the difference here is that Zamiell's guide was talking about if you wanted to ship every product, which isn't an explicit in game goal, so I agree with leaving that out here. You already provide guidance for other bundles like reminders on last day to plant crops for Summer Crops Bundle, so it seems reasonable to expect similar feedback on Quality Crops (or the need to plant a Sunflower, Poppy, and possibly Corn). Just another perspective!

Red Cabbage RNG definitely sucks, that's a whole other thread!
